pkg/services/queryhistory/queryhistory_search_test.go

```diff
@@ -0,0 +1,100 @@
+package queryhistory
```

Comment:
Needed to run the integration tests with postgres and mysql. 

```suggestion
//go:build integration
// +build integration

package queryhistory
```

Reply:
Updated in https://github.com/grafana/grafana/pull/45932/commits/7038a112e11dc11fbd24237bf34332bf8dda42fe